.github/workflows/release-deploy.yml

```diff
@@ -57,55 +64,86 @@ jobs:
         uses: actions/checkout@v4
 
       - name: 🔖 Version tag image
-        uses: aproorg/github-workflows/.github/actions/retag-docker-image@main
+        uses: aproorg/github-workflows/.github/actions/retag-docker-image@support-custom-build-scripts
         with:
           AWS_REGION: ${{ inputs.aws_region }}
           AWS_ROLE_ARN: ${{ inputs.github_ci_ecr_role_arn }}
           ECR_REPOSITORY: ${{ inputs.ecr_repository }}
+          ECR_REPOSITORIES: ${{ inputs.ecr_repositories }}
           VERSION_TAG: ${{ github.event.release.tag_name }}
 
+  unify-services:
```

Comment:
Could this be an action because it looks like the same code is also in  main-deploy?